src/gdcmodels/esmodels/case_centric/mapping.yaml

```diff
@@ -2416,6 +2416,173 @@ properties:
         type: nested
       nadir_cd4_count:
         type: double
+      other_clinical_attributes:
```

Comment:
How did you go about populating these segments?

Reply:
[This](https://github.com/NCI-GDC/gdc-models/blob/e407b50d4eb7ef8e8f0180fc4188042ca514a938/src/gdcmodels/sync/graph/common.py#L476-L478) populates the new structure in the `graph/case` & `graph/file` mappings' follow_up struct. The `case_centric` just copies whatever is found in the `graph/case` mapping.